pallets/remote-asset-manager/Cargo.toml

```diff
@@ -11,15 +11,23 @@ version = '0.0.1'
 [features]
 default = ['std']
 std = [
+    'serde',
     'codec/std',
     'frame-support/std',
     'frame-system/std',
     'xcm/std',
+    'log/std',
     'xcm-executor/std',
+    'cumulus-pallet-xcm/std',
+    'cumulus-primitives-core/std',
 ]
 # this feature is only for compilation now
 runtime-benchmarks = []
 
+[dependencies]
+log = { version = "0.4.14", default-features = false }
+serde = { version = "1.0.101", features = ["derive"], optional = true }
```

Comment:
Unnecessary imports

https://github.com/paritytech/substrate/blob/5d89967d7cc12d620bda9c9c042dbf7fcc4beb89/frame/support/src/lib.rs#L29

Reply:
serde is required in order to configure the pallet configs with genesisbuild
I'd like to keep the logs for now, because the help debugging XCM calls tremendously.